content/600-about/200-prisma-docs/20-style-guide/05-prisma-product-names.mdx

```diff
@@ -14,7 +14,6 @@ search: false
 - Prisma Migrate
 - Prisma CLI
 - Prisma Studio
-- Data Proxy (_not_ Prisma Data Proxy)
 - Data Browser (in the Prisma Data Platform UI, on cloud.prisma.io)
```

Comment:
We should remove Data Browser, Query Console and Schema Viewer as well from this list as they no longer exist.

Reply:
Nice catch! Ankur also already mentioned that so I just updated it here 😄 

https://github.com/prisma/docs/pull/5575/files#diff-907bd34b092253c949696edc5e790841fb39cad6db2a371afc6de518c7f50cde